getting_started/first_3d_game/02.player_input.rst

```diff
@@ -149,8 +149,7 @@ your interface should look like this.
 
 |image15|
 
-The final action to set up is the ``jump`` action. Bind the Space key and the gamepad's
-A button.
+The final action to set up is the ``jump`` action. Bind the Space key and joypad Button 0 (Bottom action, Sony Cross, Xbox A, Nintendo B).
```

Comment:
```suggestion
The final action to set up is the ``jump`` action. Bind the Space key and
Joypad Button 0 (Bottom Action, Sony Cross, Xbox A, Nintendo B).
```
Split up the long line, and capitalize more words, as per the original suggestion